mne/forward/_make_forward.py

```diff
@@ -549,7 +548,7 @@ def make_forward_solution(info, trans, src, bem, fname=None, meg=True,
     if fname is not None and op.isfile(fname) and not overwrite:
         raise IOError('file "%s" exists, consider using overwrite=True'
                       % fname)
-    if not isinstance(info, (dict, string_types)):
+    if not isinstance(info, (Info, string_types)):
         raise TypeError('info should be a dict or string')
```

Comment:
This here should probably be updated to ``'info should be an instance of Info or string'``.
Same with other references to dicts in the documentation.